### DISCUSSION TITLE
Firmware のReMapでの認識のさせ方について

### DISCUSSION THREAD

**Discussion by ntnl-it-wiz** [Yowkees/keyball#364] [Q&A]
Keyball61を利用しております。
標準から少し弄りたいパラメーターがあり、FirmwareのBuildからトライしております。
https://github.com/Yowkees/keyball/tree/main/qmk_firmware/keyboards/keyball

上の手順でhexファイルを作ったのち、遊舎工房さんの手順でFlashするとキーボードとしては動作はするのですが、ReMapではキーボードとして認識しませんでした。
https://yushakobo.zendesk.com/hc/ja/articles/1500011696701

認識させるために製品名の付与など必要でしょうか。

因みに私の環境では上の手順だけでは不足があり、 util/qmk_install.sh で各種ユーティリティを導入しないと動作しませんでした。
Readmeに記載がないのは環境による部分だからですかね。漏れている可能性もあるかと思いますので一応報告いたします。

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。

makeする際に、キーボード名を　keyball/keyball61:via　にするとどうでしょうか？
VIA対応ファームウェアでないとREMAPで認識しないと思います。

以上、お手数ですがご確認お願いいたします。

**Comment by ntnl-it-wiz** [comment]
回答ありがとうございます。
keyball61:viaに変更したら無事Remapで読み込めるようになりました。
